compiler2/gradle.properties

```diff
@@ -15,6 +15,5 @@
 #
 
 POM_NAME=Motif
-# TODO Update artifact id
-POM_ARTIFACT_ID=motif-compiler2
+POM_ARTIFACT_ID=motif-compiler
```

Comment:
is this artifact name changed so that compiler2 becomes the primary compiler when pulling in motif?

Reply:
The new implementation used to use the `compiler2` artifact id so it could exist alongside the old compiler on Maven Central. We're not deleting the old compiler completely so we can use the `compiler` artifact id.